apps/account-api/src/main.ts

```diff
@@ -33,17 +33,11 @@ function startShutdownTimer() {
 }
 
 async function bootstrap() {
-  process.on('uncaughtException', (error) => {
```

Comment:
I'm unsure why this is here; it's not in all the apps main.ts, and when I was testing error output using just a random throw call, this never gets tripped because of the try/catch at the end.  :woman_shrugging: